src/styles/blocks/days.scss

```diff
@@ -0,0 +1,32 @@
+@import "../utils/variables";
+
+.day {
+  width: $cell-width;
+  height: $cell-width;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid $border-color;
+  background-color: $colors-square;
+  transition: 0.5s;
```

Comment:
don't animate all, specify properties you need

Reply:
still not fixed